microcosm_flask/conventions/crud.py

```diff
@@ -54,9 +54,9 @@ def configure_search(self, ns, definition):
         @qs(definition.request_schema)
         @response(paginated_list_schema)
         def search(**path_data):
-            page = self.page_cls.from_query_string(definition.request_schema)
-            result = definition.func(**merge_data(path_data, page.to_dict(func=identity)))
-            response_data, headers = page.to_paginated_list(result, ns, Operation.Search)
```

Comment:
A key detail you are missing here: we can (and do!) override `__str__` in enums.

When we build query string arguments for links, [we provide a func](https://github.com/globality-corp/microcosm-flask/blob/develop/microcosm_flask/paging.py#L155), defaulting to `str` to convert values to something representable in our code. We expect user-provided `Enums` to specify an appropriate `__str__` function.

My guess is that you can avoid this PR entirely by adding `__str__` in the right places?